examples/tutorials/advanced/cartesian_histograms.py

```diff
@@ -204,10 +205,8 @@
     frame=["wSnE", "xaf10", "ya5f1+lCumulative counts"],
     data=data01,
     series=10,
-    # Use pattern ("p") number 8 as fill for the bars
-    # Set the background ("+b") to white [Default]
-    # Set the foreground ("+f") to black [Default]
-    fill="p8+bwhite+fblack",
+    # Fill bars with GMT pattern 8, with white background and black foreground.
+    fill=Pattern(8, bgcolor="white", fgcolor="black"),
```

Comment:
At L149 above, could you add an intersphinx link to ```:class:`pygmt.params.Pattern` ```? Then we can properly close #2323.

Reply:
You meant L179, right? Done in 5774b35254e9447539bd9b38a3d8df84bc461de4.